Source/Core/VideoBackends/Vulkan/VulkanLoader.cpp

```diff
@@ -36,6 +36,7 @@ static void ResetVulkanLibraryFunctionPointers()
 #define VULKAN_INSTANCE_ENTRY_POINT(name, required) name = nullptr;
 #define VULKAN_DEVICE_ENTRY_POINT(name, required) name = nullptr;
 #include "VideoBackends/Vulkan/VulkanEntryPoints.inl"
+
```

Comment:
Same comment as my other, there are no changes to VulkanLoader.cpp other than formatting.